diff --git a/configure b/configure
index 8eb25346e..f9058e0ac 100755
--- a/configure
+++ b/configure
@@ -12,13 +12,5 @@ inpath()
     return 1
 }
 
-for p in python3 python3.11 python3.10 python3.9 python3.8 python3.7 python3.6 python
-do
-    if ( inpath $p ); then
-        exec $p `dirname $0`/make/configure.py "$@"
-        exit 0
-    fi
-done
-
-echo "ERROR: no suitable version of python found."
-exit 1
+exec python `dirname $0`/make/configure.py "$@"
+exit 0